### PR TITLE
convert competition entries to articles

### PR DESCRIPTION
**Pull request by codiebeulaine**
Create action that saves the entry as a draft article on wagtail

### CODE REVIEW

**Comment by codiebeulaine** [comment]
@miltontony keep getting an error with wagtail, review please?

**Comment by miltontony** [comment]
What error?

File: molo/yourwords/admin.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from molo.core.models import ArticlePage
+from molo.yourwords.models import YourWordsCompetitionEntry
+from django.contrib import admin
+
+
+def ConvertToArticle(model_admin, request, entry):
+    article = ArticlePage.objects.create(null, null)
```
Comment (miltontony) [inline molo/yourwords/admin.py]: So...not quite how creating pages works in Wagtail..There's a tree structure and all pages need to be added in a tree. Have a look at [these tests](https://github.com/praekelt/molo/blob/develop/molo/core/tests/test_models.py#L52-L71) for an example. According to the SoW, the new article will need to be added to a Section called "True Story" or something.. Let's chat in slack

Comment (miltontony) [inline molo/yourwords/admin.py]: Just another note that creating an article is not enough, you'll need to save it as a draft which looks something like this

```
article.save_revision(
    user=request.user,
    submitted_for_moderation=False,
)
```

**Comment by codiebeulaine** [comment]
@milton ready for review? Just removed an unessecary (don't know how to spell this) line

File: molo/yourwords/tests/fixtures/test.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[{
```
Comment (miltontony) [inline molo/yourwords/tests/fixtures/test.json]: I don't think you need this fixture at all anymore. You can just create a user normally.

**Comment by codiebeulaine** [comment]
@miltontony removed fixtures, still passing. :+1: ?

**Comment by miltontony** [comment]
:+1:  This is the first part - the second part should be a redirect to the "move" page. Should be in another PR

**Pull request merged**